ui/src/views/Tasks/index.jsx

```diff
@@ -84,11 +81,13 @@ export default class Task extends Component {
         />
         <RouteWithProps
           path={`${path}/:taskId/runs/:runId/logs/live/:logUrl`}
+          stream
           {...props}
-          component={LiveTaskLog}
+          component={TaskLog}
         />
         <RouteWithProps
           path={`${path}/:taskId/runs/:runId/logs/:logUrl`}
+          stream={false}
```

Comment:
`stream` is `false` by default. We can remove this line.

Reply:
Yeah. I was wondering whether it would be obligatory to show the same.